modules/kubernetes-addons/keda/variables.tf

```diff
@@ -10,6 +10,12 @@ variable "manage_via_gitops" {
   description = "Determines if the add-on should be managed via GitOps"
 }
 
+variable "iam_role_path" {
+  type        = string
+  default     = "/"
+  description = "IAM role path"
+}
+
```

Comment:
This variable already exists

Reply:
this needs to be removed

Reply:
Removed it now. Didn't see a conflict in the rebase, apologies.